test/dotnet-remove-proj.Tests/GivenDotnetRemoveProj.cs

```diff
@@ -273,5 +316,82 @@ public void WhenPassedMultipleReferencesAndOneOfThemDoesNotExistItRemovesTheOneT
             slnFile.Projects.Count.Should().Be(1);
             slnFile.Projects[0].FilePath.Should().Be(Path.Combine("App", "App.csproj"));
         }
+
+        [Fact]
+        public void WhenReferenceIsRemovedBuildConfigsAreAlsoRemovedAndSlnBuilds()
```

Comment:
But then most of that code is duplicated? I wanted an end-to-end type test where I can validate the operation but also make sure it builds properly. I don't know how to break this one up.